Kzq test

Kkkzq opened the pull request:
Webpage design.

Kkkzq committed:
modified:   insert.php

Kkkzq committed:
modified:   insert.php
	deleted:    inventory.php
	deleted:    search.inc.php

esjoo commented:
Bootstrap and Jquery is already fetched by the clients browser when running header.php, add a link to angular today. When doing CSS please refrain from setting styles on div since its used by many other classes.  Otherwise goodjob!

Kkkzq committed:
modified:   README.md
	modified:   calendar.php
    modified:   db.php
	modified:   footer.php
	modified:   header.php
	modified:   home.php
	modified:   index.php
	modified:   insert.inc.php
	modified:   insert.php
	modified:   login.inc.php
	modified:   login.php
	modified:   signup.php
	account.php
	add_to_inventory.php
	ajaxFORinsert.php
	bootstrap-select.min.css
	bootstrap-select.min.js
	bootstrap.min.css
	bootstrap.min.js
	chemicalCalculation.php
	create_database.txt
	create_db_NEW.txt
	experimentModal.php
	gfx/
	includes/
	inventory.php
	jquery.min.js
	livesearchFORmanageUsers.php
	livesearchFORprotocol.php
	logout.php
	manageUsers.php
	newExperiment.php
	protocolList.php
	query_for_insertChemical.php
	remove_exp.php
	scripts/
	search.inc.php
	showProtocol.php
	sidebarCalendar.php
	style/
